gossip/src/cluster_info.rs

```diff
@@ -12,6 +12,7 @@
 //! * layer 2 - Everyone else, if layer 1 is `2^10`, layer 2 should be able to fit `2^20` number of nodes.
 //!
 //! Bank needs to provide an interface for us to query the stake weight
+pub use solana_net_utils::{MINIMUM_VALIDATOR_PORT_RANGE_WIDTH, VALIDATOR_PORT_RANGE};
```

Comment:
Let's add that deprecation warning too

Reply:
Added. Not really sure which version to use, so I just set it as deprecated since 1.10.0